### PR TITLE
Remove port, username and password from ES variables

### PR DESCRIPTION
**Pull request by rsevilla87**
Depends on: https://github.com/cloud-bulldozer/benchmark-operator/pull/438

Will also require some changes in https://github.com/openshift-scale/scale-ci-pipeline

Signed-off-by: Raul Sevilla <rsevilla@redhat.com>

### CODE REVIEW

File: workloads/kube-burner/README.md [hunk 1]
```diff
@@ -15,7 +15,6 @@ All scripts can be tweaked with the following environment variables:
 | **QPS**              | Queries/sec                     | 10      |
 | **BURST**            | Burst queries                   | 10      |
 | **ES_SERVER**        | Elastic search endpoint         | https://search-cloud-perf-lqrf3jjtaqo7727m7ynd2xyt4y.us-west-2.es.amazonaws.com|
```
Comment (jtaleric) [inline workloads/kube-burner/README.md:17]: this should be `https/http/g`

File: workloads/kube-burner/common.sh [hunk 2]
```diff
@@ -3,7 +3,6 @@ export CERBERUS_URL=${CERBERUS_URL}
 export QPS=${QPS:-10}
 export BURST=${BURST:-10}
 export ES_SERVER=${ES_SERVER:-https://search-cloud-perf-lqrf3jjtaqo7727m7ynd2xyt4y.us-west-2.es.amazonaws.com}
```
Comment (jtaleric) [inline workloads/kube-burner/common.sh:5]: `https/http/g`

**Review by jtaleric — CHANGES_REQUESTED** [review]
Since we are referencing the public ES server, lets be sure to call out `http` vs `https` as if we reference `https` we should call out the port as 443 vs 80.

**Review by jtaleric — APPROVED** [review]
LGTM!

**Review by whitleykeith — APPROVED** [review]

**Review by mohit-sheth — APPROVED** [review]
LGTM

**Pull request merged**